src/main/java/frc/robot/subsystems/drive/Drivetrain.java

```diff
@@ -234,16 +318,17 @@ private void move(Vector spdVec, double rot) {
    * @param rateLimit whether or not to use slew rate limiting
    */
   private void move(Vector spdVec, double rot, boolean rateLimit) {
-    Vector spdCommanded = spdVec;
+    spdVec = spdVec.copy();
+
     m_currentRotationRadians = rot;
 
-    if (rateLimit) {
+    if (/*rateLimit*/false) {
```

Comment:
Why is this commented out? Rate limiting should not be removed.

Reply:
@garrettwitowski Something in the slew rate limiting code is causing the drivetrain code to malfunction (stalling and not responding to joystick input).

Reply:
@Irishumanoid then we need to debug that before we merge this. This was not the case on Saturday and that codepath was active.

Reply:
Got it